storage-api/src/main/java/com/linagora/calendar/storage/AlarmEvent.java

```diff
@@ -0,0 +1,31 @@
+/********************************************************************
+ *  As a subpart of Twake Mail, this file is edited by Linagora.    *
+ *                                                                  *
+ *  https://twake-mail.com/                                         *
+ *  https://linagora.com                                            *
+ *                                                                  *
+ *  This file is subject to The Affero Gnu Public License           *
+ *  version 3.                                                      *
+ *                                                                  *
+ *  https://www.gnu.org/licenses/agpl-3.0.en.html                   *
+ *                                                                  *
+ *  This program is distributed in the hope that it will be         *
+ *  useful, but WITHOUT ANY WARRANTY; without even the implied      *
+ *  warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR         *
+ *  PURPOSE. See the GNU Affero General Public License for          *
+ *  more details.                                                   *
+ ********************************************************************/
+
+package com.linagora.calendar.storage;
+
+import java.time.Instant;
+
+import org.apache.james.core.MailAddress;
+
+import com.linagora.calendar.storage.eventsearch.EventUid;
+
+public record AlarmEvent(EventUid eventId,
+                         Instant alarmTime,
```

Comment:
Per discussion with @hungphan227, 
we will also store:
- Instant eventStartTime 
- boolean recurrenceEvent. 

Then we done all the cases.